extensions/amp-video/1.0/video-wrapper.js

```diff
@@ -114,6 +131,46 @@ export function VideoWrapper({
     };
   }, [mediasession, playing, metadata, play, pause]);
 
+  useImperativeHandle(
+    ref,
+    () =>
+      Object.defineProperties(
+        {
+          // Standard HTMLMediaElement/Element
+          play,
+          pause,
+          requestFullscreen,
+
+          // Non-standard
+          userInteracted,
+          mute: () => setMuted(true),
+          unmute: () => {
+            if (hasUserInteracted) {
+              setMuted(false);
+            }
+          },
+        },
+        {
+          // Standard HTMLMediaElement/Element
+          currentTime: {get: () => playerRef.current.currentTime},
+          duration: {get: () => playerRef.current.duration},
+          autoplay: {get: () => autoplay},
```

Comment:
Are `controls` and `autoplay` just passthrough from props? Are they here because they are also in the MediaElement API?

Reply:
Yes and yes :)